authors/apps/articles/tests/test_create_article.py

```diff
@@ -206,3 +208,71 @@ def test_to_fetch_non_exisiting_article(self):
                                    format='json')
         self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
         self.assertEqual('Not found.', response.data['detail'])
+
+    def test_favorite_article(self):
+        """Test to favorite an article"""
+        token = self.signup_user_one()
+        del self.create_article_data['images']
+        self.client.post(self.article_listcreate,
```

Comment:
I'm sure you don't want us to focus on this part.. so create a helper function that creates an article so you can call it in one line and we can focus on the favorite endpoint below.

Reply:
Done, I used the helper functions elsewhere to reduce duplication